src/legacy/Routes.tsx

```diff
@@ -111,7 +112,7 @@ class RouteApp extends React.Component<any, any> {
     this.rpc = rpc
 
     // Auto refresh every 10s
-    if (!app?.isPackaged) {
+    if (process.env.NODE_ENV === 'production') {
```

Comment:
I think this code is not correct, We need to use app.isPackaged. See sample here: https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/features/loading/LoadingScreen.tsx (Line 49)